apps/fluent-tester/src/FluentTester.tsx

```diff
@@ -53,31 +59,62 @@ const TestListSeparator = Separator.customize((t) => ({
   separatorWidth: 2,
 }));
 
-export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props: FluentTesterProps) => {
-  // filters and sorts tests alphabetically
-  const filteredTestComponents = tests.filter((test) => test.platforms.includes(Platform.OS as string));
-  const sortedTestComponents = filteredTestComponents.sort((a, b) => a.name.localeCompare(b.name));
+const Header: React.FunctionComponent<HeaderProps> = (props) => {
+  const { enableSinglePaneView, enableBackButtonIOS, onBackButtonPressedIOS } = props;
+  const theme = useTheme();
 
-  const { initialTest, enableSinglePaneView } = props;
-  const initialSelectedTestIndex = sortedTestComponents.findIndex((description) => {
-    return description.name === initialTest;
-  });
+  const headerStyle = enableSinglePaneView ? fluentTesterStyles.headerWithBackButton : fluentTesterStyles.header;
+
+  return (
+    <View style={headerStyle}>
+      <Text
+        style={fluentTesterStyles.testHeader}
+        variant="heroLargeSemibold"
+        color={theme.host.palette?.TextEmphasis}
+        testID={BASE_TESTPAGE}
+      >
+        ⚛ FluentUI Tests
+      </Text>
+      <View style={fluentTesterStyles.header}>
+        {/* On iPhone, We need a back button. Android has an OS back button, while desktop platforms have a two-pane view */}
+        {Platform.OS === 'ios' && !Platform.isPad && (
+          <Button
+            appearance="subtle"
+            style={fluentTesterStyles.backButton}
+            onClick={onBackButtonPressedIOS}
+            disabled={!enableBackButtonIOS}
+          >
+            ‹ Back
```

Comment:
super nit: RTL?

Reply:
Hmm.. Let me try. This might be one of those RTL friendly characters. Let's see. 
But also, none of our strings are localized... so LTR is kinda implied...

Reply:
I shall go fix! 
![image](https://user-images.githubusercontent.com/6722175/207172770-6b71ba5e-4337-48b8-91f3-33a948e22dac.png)

Reply:
you might be ok if you remove the chevron and have it just say "back"? Or lose the text and only have an icon?

Reply:
Fixed! 

![Screenshot 2022-12-12 at 3 00 58 PM](https://user-images.githubusercontent.com/6722175/207174780-e7b4e537-e93e-4506-bcea-f32be7b0bc05.png)

Reply:
Ah, noticed your comment about losing the chevron. Hmm.. I think I'll keep it as is as an example of caring about RTL.  I think I would prefer Icon only overall, but would want to make it bigger / bolder. I kinda wanna revisit the header overall but that's out of scope of this PR.